opportunities/index.html

```diff
@@ -21,16 +21,16 @@
           <h3 >Researchers and Developers</h3>
         </header>
 
-        <!--<p style="padding-bottom: 40px;  ">We are currently not hiring. Please check back later for more info.</p-->
-	<p>We are looking for several highly-motivated graduate and upper-level undergraduate students to join our team. For posting information and instructions how to apply, please consult the postings on the ASU student employment website. Please do not send individual emails to us. We can only consider applications submitted through the ASU student employment site. Simply follow the instructions below: </p>
-	<ol style="list-style-type: decimal; padding-left: 15px;">
+        <p style="padding-bottom: 40px;  ">We are currently not hiring. Please check back later for more info.</p>
+	<!--<p>We are looking for several highly-motivated graduate and upper-level undergraduate students to join our team. For posting information and instructions how to apply, please consult the postings on the ASU student employment website. Please do not send individual emails to us. We can only consider applications submitted through the ASU student employment site. Simply follow the instructions below: </p-->
+	<!--<ol style="list-style-type: decimal; padding-left: 15px;">
 
 		<li>Visit <a href="https://students.asu.edu/employment/search" target="_blank">https://students.asu.edu/employment/search</a>.</li>
 		<li>Click on "Search On-Campus Jobs".</li>
 		<li>Click on "Search for Openings".</li>
 		<li>Search for requisition ID <strong>97498BR</strong>.</li>
 		<li>Follow the instructions listed.</li>
-	</ol>
+	</ol-->
```

Comment:
should be a proper comment ending, `</ol>-->

Reply:
fixed